mobly/controllers/android_device_lib/adb.py

```diff
@@ -537,22 +538,30 @@ def root(self) -> bytes:
 
     Raises:
       AdbError: If the command exit code is not 0.
+      AdbTimeoutError: If the command timed out.
     """
     retry_interval = ADB_ROOT_RETRY_ATTEMPT_INTERVAL_SEC
     for attempt in range(ADB_ROOT_RETRY_ATTEMPTS):
       try:
         return self._exec_adb_cmd(
-            'root', args=None, shell=False, timeout=None, stderr=None
+            'root',
+            args=None,
+            shell=False,
+            timeout=ADB_ROOT_ATTEMPT_TIMEOUT_SEC,
+            stderr=None,
         )
-      except AdbError as e:
+      except (AdbError, AdbTimeoutError) as e:
         if attempt + 1 < ADB_ROOT_RETRY_ATTEMPTS:
           logging.debug(
-              'Retry the command "%s" since Error "%s" occurred.'
+              'Retry the command "%s" since %s.'
```

Comment:
When something don't fit in one line, it's better to use multiple lines for better readability